resolver/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ockam-network/did"
+	"github.com/ockam-network/ockam"
+	"github.com/ockam-network/ockam/node"
+	ockamHttp "github.com/ockam-network/ockam/node/remote/http"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func run() error {
+	mux := http.NewServeMux()
+
+	port := os.Getenv("PORT")
+	discoverer_name := os.Getenv("DISCOVERER_NAME")
+	discoverer_port, err := strconv.Atoi(os.Getenv("DISCOVERER_PORT"))
+
+	if err != nil {
+		exitOnError(err)
+	}
+
+	mux.Handle(fmt.Sprintf("/%s/identifiers/ockam/", Version()), handleGetEntity(discoverer_name, discoverer_port))
+
+	log.Println(fmt.Sprintf("/%s/identifiers/ockam/", Version()))
+	log.Println(fmt.Sprintf("Listening on %s\n", port))
+
+	if err := http.ListenAndServe(fmt.Sprintf(":%s", port), mux); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func handleGetEntity(name string, port int) http.Handler {
```

Comment:
it would ideal to bubble up all errors up to main ... so main this return `http.Handler, error`